<ui version="4.0" >
 <class>udg::Q2DViewerExtensionBase</class>
 <widget class="QWidget" name="udg::Q2DViewerExtensionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1567</width>
    <height>438</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_buttonGrid" >
         <property name="toolTip" >
          <string>Series layout</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/grid.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_downButtonGrid" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Series layout</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/down.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_imageGrid" >
         <property name="toolTip" >
          <string>Image layout</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/grid.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_downImageGrid" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Image layout</string>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/down.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_slicingToolButton" >
         <property name="text" >
          <string>Slicing</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/slicing.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_windowLevelToolButton" >
         <property name="text" >
          <string>window level</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/windowLevel.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_translateToolButton" >
         <property name="text" >
          <string>move</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/move.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_zoomToolButton" >
         <property name="text" >
          <string>zoom</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/zoom.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_oldDistanceToolButton" >
         <property name="text" >
          <string>distance</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/distance.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_polylineButton" >
         <property name="text" >
          <string>zoom</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/polyline.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_distanceToolButton" >
         <property name="text" >
          <string>distance</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/distance.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_roiToolButton" >
         <property name="text" >
          <string>ROI</string>
         </property>
         <property name="icon" >
          <iconset/>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_eraserToolButton" >
         <property name="text" >
          <string>distance</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/eraser2.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_axialViewToolButton" >
           <property name="text" >
            <string>Axial</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/axial.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_sagitalViewToolButton" >
           <property name="text" >
            <string>Sagital</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/sagital.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_coronalViewToolButton" >
           <property name="text" >
            <string>Coronal</string>
           </property>
           <property name="icon" >
            <iconset resource="../../../main/main.qrc" >:/images/coronal.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Orientation</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCINEController" native="1" name="m_cineController" />
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_viewerInformationToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/viewerInfo.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_voxelInformationToolButton" >
         <property name="text" >
          <string>voxel information</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/voxelInformation.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_referenceLinesToolButton" >
         <property name="text" >
          <string>Reference Lines</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_dicomDumpToolButton" >
         <property name="text" >
          <string>Dicom dump</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/info.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_screenShotToolButton" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/photo.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_presentationStateSwitchToolButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>PS</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>WW/WL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWindowLevelComboBox" name="m_windowLevelComboBox" >
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_rotateClockWiseToolButton" >
         <property name="text" >
          <string>+90</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/rotateClockWise.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_rotateCounterClockWiseToolButton" >
         <property name="text" >
          <string>-90</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/rotateCounterClockWise.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_flipHorizontalToolButton" >
         <property name="text" >
          <string>FlipH</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/flipHorizontal.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_flipVerticalToolButton" >
         <property name="text" >
          <string>FlipV</string>
         </property>
         <property name="icon" >
          <iconset resource="../../../main/main.qrc" >:/images/flipVertical.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="m_workingArea" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QWindowLevelComboBox</class>
   <extends>QComboBox</extends>
   <header>qwindowlevelcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QCINEController</class>
   <extends>QWidget</extends>
   <header>qcinecontroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../main/main.qrc" />
  <include location="q2dviewer.qrc" />
 </resources>
 <connections/>
</ui>
